### DISCUSSION TITLE
KINDLY HELP....! Expected to find element: #VTK-j_idt1363, but never found it.

### DISCUSSION THREAD

**Discussion by razeemmansoor** [cypress-io/cypress#20640] [General]
Hi..,I am facing an issue on clicking an element.I need an element to be clicked while the network is slow by not using wait() and pause().The page loads after or more than 35sec(350000).Could someone help me to solve the issue,please?

Here is my coding.

it('Test ship',function(){

    cy.viewport(1120,800)

 
     cy.visit('url')
 
     cy.get('#LoginForm-nfr_login_authname').type('username')
 
     cy.get('#LoginForm-nfr_login_authid').type('paSSw0rd')
 
     cy.on('uncaught:exception', (err, runnable) => {
         return false
     })
     

     //These two were the elements present in the same page.The Workspace (first command) gets loaded and error occurs before clicking the second element.

      cy.wait[ cy.xpath('//span[normalize-space()="Work Space"]') .click({force: true})]
      
     cy.wait[ cy.get('#VTK-j_idt1262').click({force: true})]

    })

Please help me to sort out this problem.

**Comment by jjhelguero** [comment]
I don't think that is the proper use of a  `cy.wait()` command. It seems you are waiting for an element to be rendered/enabled. I'd recommend the following and add assertions such as `.should('be.visible')` or `.should('be.enabled')` where you may need.

```
it('Test ship', function() {
  cy.viewport(1120,800)
  cy.visit('url')
  cy.get('#LoginForm-nfr_login_authname').type('username') // maybe use better selectors?
  cy.get('#LoginForm-nfr_login_authid').type('paSSw0rd')
  cy.on('uncaught:exception', (err, runnable) => { // you are allowing all app errors to occur, might be harmful to app quality
       return false
  })

  cy.xpath('//span[normalize-space()="Work Space"]').should('be.visible').click() // try to use better selectors than xpath
  cy.get('#VTK-j_idt1262').should('be.visible').click()
})
```

**Comment by razeemmansoor** [comment]
But I have used that assertions also,still unable to click the element..I
tried many possibilities already available in the internet.Can you please
suggest me some more possibilities

**Reply by jjhelguero** [reply]
Maybe try cypress-real-events `.realClick()`

**Reply by razeemmansoor** [reply]
ok.let me try

**Reply by razeemmansoor** [reply]
no this too doesnt works.
